.github/renovate/default.json5

```
@@ -20,13 +20,12 @@
     },
     customDatasources: [
         "nextflow-plugin": {
-            "defaultRegistryUrlTemplate": "https://api.github.com/repos/nextflow-io/{{pluginName}}/releases"
+            "defaultRegistryUrlTemplate": "https://raw.githubusercontent.com/nextflow-io/plugins/main/plugins.json"
             "format": "json",
             "versioning": "semver",
             "transformTemplate": [
-                "{\"release\": \"{{version}}\", \"releaseTimestamp\": \"{{date}}\"}"
+                "$[id = 'nf-validation']"
```

```suggestion
                "$[id = {{pluginName}}]"
```
would like to keep it not plugin specific (nf-prov will probably soon also be in the template)